status/templates/status/statusTable.html

```diff
@@ -30,8 +30,7 @@
           "success" {% else %} "danger" {% endif %}>
           <td>
             {% if submission.grouper|show_detail:user %}
-              <a target="_blank"
```

Comment:
We can disable it. If an user requires new tab, they can "open it in a new tab".